_posts/2019-08-13-how-to-deploy-production-grade-vpc-aws.adoc

```diff
@@ -1025,15 +1048,43 @@ You'll also need to update the NACLs to allow access from the management VPC:
 [source,hcl]
 ----
 module "vpc_network_acls" {
-  source = "git@github.com:gruntwork-io/module-vpc.git//modules/vpc-app-network-acls?ref=<VERSION>"
+  source = "git::git@github.com:gruntwork-io/terraform-aws-vpc.git//modules/vpc-app-network-acls?ref=<VERSION>"
 
   # ... (other params omitted) ...
 
   allow_access_from_mgmt_vpc = true
-  mgmt_vpc_cidr_block        = data.terraform_remote_state.mgmt_vpc.vpc_cidr_block
+  mgmt_vpc_cidr_block        = data.terraform_remote_state.mgmt_vpc.outputs.vpc_cidr_block
+}
+----
+
+# TODO: The outputs include `dns_mgmt_to_app` but that module is not defined for the main.tf above. Since the outputs are linked to main
+branch, and main.tf is hardcoded, there's a mismatch in versions.
+# it's here: https://github.com/gruntwork-io/infrastructure-modules-multi-account-acme/blob/master/networking/vpc-app/main.tf#L183-L218
+
+You'll also need to create a DNS forwarder between the management VPC and the app VPC:
```

Comment:
We actually want to drop this (the resources this module deploys is pretty expensive, and is only useful for a very narrow use case), so what we really should do is update the acme code to reflect this guide.

Reply:
Ah, that works for me. I had to delete all the mgmt VPC resources anyway, because I hit limits when deploying the App VPC afterward. I'll go ahead and also delete my mgmt VPC now.

Thanks for the note about it.

Reply:
Hold on--- Do we want to remove mgmt VPC stuff? And the peering connection? Or just the dns forwarder? DNS forwarder gets added in the EKS guide. We need that? Or we don't need it even for EKS?

Reply:
No we want to keep the peering, but the DNS forwarder is no longer necessary. Since writing the EKS guide, AWS solved the problem where we needed the DNS forwarder such that it is moot (it is still relevant for resolving Private route 53 DNS entries, but that is a rarer use case).

Reply:
Do we still need DNS forwarding to make the EKS control plane accessible to another VPC?

Reply:
Removed this section assuming that was all that was needed.